src/pages/RFOX/components/Overview/Overview.tsx

```diff
@@ -0,0 +1,28 @@
+import { Card, CardBody, CardHeader, Flex } from '@chakra-ui/react'
+import { foxAssetId } from '@shapeshiftoss/caip'
+import { AssetIcon } from 'components/AssetIcon'
+import { RawText } from 'components/Text'
+
+import { Positions } from './Positions'
+import { Totals } from './Totals'
+
+export const Overview: React.FC = () => {
+  return (
+    <Card>
+      <CardHeader pt={6} borderBottomWidth={1} borderColor='border.base'>
+        <Flex alignItems='center' gap={2} mb={6}>
+          <AssetIcon size='sm' assetId={foxAssetId} key={foxAssetId} showNetworkIcon={false} />
```

Comment:
I did move it to a variable but I wan't to avoid pushing it through a prop as I feel we would just need to use a reusable hook there and keep this component composable

Reply:
Yeah that's absolutely fine! As mentioned no need to pass it over in props for now, let's do that as part of the wire-up